homework_2/task_1/internal/infrastructure/http2/axilisOfferFeed2.go

```diff
@@ -0,0 +1,118 @@
+package http2
```

Comment:
So, you are saying that I should only avoid using numbers while naming packages, since you suggested that name of my second feed structure has a number?

Reply:
Yes and no. 
There is a subtle difference between these cases and like all rules there are exceptions. I would argue that `http2` doesn't carry much information more information than `http` (where structs which use http reside), while `AxilisOfferFeed2` is named like the feed url, which this component is coupled to. In that case the number in the name corresponds to the domain, since that is the way it is defined.
Of course, these are just suggestions, feel free to name things how you prefer.